File: Day8/Solution.py
n = int(raw_input())
phonebook = {}
for i in range(0,n):
    name, id = raw_input().split()
    phonebook[name] = id
while (1):
    temp = ''
    name = raw_input()
    if (name == temp):
        break
    if(phonebook.has_key(name)):
        print name + "=" + phonebook[name]
    else:
        print "Not found"

File: Day8/Solution.cpp
int main() {
    int n;
    cin >>n;
    map<string, string> phonebook;
    string name, id;
    for (int i=0; i<n;i++) {
      cin>>name >> id;
      phonebook[name] = id;
    }
    while(cin >> name) {
    if (phonebook.find(name) != phonebook.end()) {
        cout << name << "=" << phonebook.find(name)->second << endl;
    } else {
        cout << "Not found" << endl;
    }
}

    return 0;
}

